### DISCUSSION TITLE
Laravel 10 Newsletter does nothing on mailchimp

### DISCUSSION THREAD

**Discussion by gbelvedere** [spatie/laravel-newsletter#324] [Q&A]
Installed by the instructions, I get nothing in mailchimp. 

the .env file is like this:

```
NEWSLETTER_ENDPOINT=
NEWSLETTER_API_KEY=XXX
NEWSLETTER_LIST_ID=YYY
```

config file with default content except for: 
    'driver' => env('NEWSLETTER_DRIVER', Spatie\Newsletter\Drivers\MailChimpDriver::class),

Any method I tried returns false and does nothing in mailchimp. 
Tried putting wrong API or list ID and it complains about it, so seems to be working

**Comment by tom-sk** [comment]
I thought I was having this issue as well, but I think mailchimp might of been blocking my test emails?

**Comment by YaraElmalah** [comment]
This happened to me as well. In the end, I worked directly with the MailChimp guide.
